### PR TITLE
New methods

### PR DESCRIPTION
**Pull request by ArtemVakulenko**

### CODE REVIEW

**Review by MarynaHrushevska — APPROVED** [review]

File: src/backend/connectors/Neo4jConnector.js [hunk 1]
```diff
@@ -72,7 +72,24 @@ class Neo4jConnector extends BaseConnector{
...
+    putPersonBack(personId, func) {
+        super.putPersonBack(personId, func)
+        this.#query(`MATCH (n) WHERE id(n) = ${person.id} SET n.deleted = 0`, ()=>{})
+    }
+
+    deletePersonsByUserId(user_id, func) {
+        super.deletePersonsByUserId(user_id, func)
+        this.#query(`MATCH (n) WHERE n.user_id = ${user_id} SET n.deleted = 1`, ()=>{})
+    }
+
```
Comment (RuslanYaremenkoWizardsDev) [inline src/backend/connectors/Neo4jConnector.js:86]: remove empty line

File: src/backend/connectors/Neo4jConnector.js [hunk 2]
```diff
@@ -72,7 +72,24 @@ class Neo4jConnector extends BaseConnector{
...
+
+    putPersonsBackByUserId(user_id, func) {
+        super.putPersonsBackByUserId(user_id, func)
+        this.#query(`MATCH (n) WHERE n.user_id = ${user_id} SET n.deleted = 0`, ()=>{})
+    }
+
```
Comment (RuslanYaremenkoWizardsDev) [inline src/backend/connectors/Neo4jConnector.js]: remove empty line

File: src/backend/connectors/SQLIteConnector.js [hunk 3]
```diff
@@ -33,7 +33,7 @@ class SqliteConnector extends BaseConnector{
 
     postPerson(person, func) {
       super.postPerson(person, func);
-      this.#connection.run(`INSERT INTO persons (fname, lname, age, city, phonenumber, email, companyname, user_id, deleted) VALUES ('${person.fname}', '${person.lname}', ${person.age}, '${person.city}', '${person.phoneNumber}', '${person.email}', '${person.companyName}', '${person.user_id}', FALSE)`, func);
+      this.#connection.run(`INSERT INTO persons (fname, lname, age, city, phonenumber, email, companyname, user_id, deleted) VALUES ('${person.fname}', '${person.lname}', ${person.age}, '${person.city}', '${person.phoneNumber}', '${person.email}', '${person.companyName}', ${person.user_id}, FALSE)`, func);
```
Comment (RuslanYaremenkoWizardsDev) [inline src/backend/connectors/SQLIteConnector.js:37]: the same

File: src/backend/connectors/SQLIteConnector.js [hunk 4]
```diff
@@ -52,13 +52,32 @@ class SqliteConnector extends BaseConnector{
...
+      super.deletePersonsByUserId(userId, func)
+ 
+        this.#connection.run(`UPDATE persons SET deleted=1 WHERE user_id=${userId}`, func)
+    }
+  
+    putPersonsBackByUserId (userId, func) {
+      super.putPersonsBackByUserId(userId, func)
+
+        this.#connection.run(`UPDATE persons SET deleted=0 WHERE user_id=${userId}`, func)
+
```
Comment (RuslanYaremenkoWizardsDev) [inline src/backend/connectors/SQLIteConnector.js]: line

**Review by DanielKubariev — APPROVED** [review]
4etko

**Review by DanielKubariev — APPROVED** [review]

**Review by DanielKubariev — APPROVED** [review]

**Pull request merged**